src/cgame/cg_gameinfo.cpp

```diff
@@ -41,21 +41,20 @@ static std::vector<arenaInfo_t> arenaList;
 CG_ParseInfos
 ===============
 */
-static void CG_ParseInfos( const char *buf )
+static void CG_ParseInfos( std::string text )
 {
-	char *token;
-	char key[ MAX_TOKEN_CHARS ];
+	const char *buf = text.c_str();
 
 	while ( true )
 	{
-		token = COM_Parse( &buf );
+		std::string token = COM_Parse( &buf );
```

Comment:
Well, it makes possible to use easier to read syntax like `str == '"something"` or `str.empty()` instead of a bunch of `strcmp(str, "something") == 0` and others. This is just making code looking better.

Reply:
Yes. I think std::string_view is a better way to handle this. std::string makes a copy every time you assign something to it.

Reply:
It looks like `std::string_view` is C++17, do we have that? On my end it doesn't build.

https://en.cppreference.com/w/cpp/string/basic_string_view